protected_app_signals_cost.md

```diff
@@ -0,0 +1,163 @@
+*   The cost for the SFE of sending the SelectAdResponse response to the Ad Server (see [C8 in the B&A cost explainer][9])
+
+In the case of PAS the costs C1 and C2 of the communication from AdServer to SFE will depend on the solution adopted for ad retrieval In case of TEE ad retrieval the RTB server will send to the Ad Server information to help during retrieval and bidding such as contextual embeddings while in case of contextual the amount of data sent is expected to be higher since it will contain an additional list of ad identifiers and possibly some per-ad embeddings to be used during TopK selection (in case of hybrid ad retrieval) or bidding.
+
+### H, V: Cost of running the SFE and Auction Server
+
+The CPU cost of running both the SFE and Auction Server are expected to be similar to that of [serving the PA traffic][10].
+
+### T: Cost associated to the traffic from SFE to the K/V Server
+
+This cost too is expected to be the [same as per the PA traffic][5].
+
+Buyer Cost
+----------
+
+We will refer to [this breakdown][11] from the B&A cost explainer.
+
+### D,E: Cost associated to the extra traffic from the Seller Ad Server to the RTBs
+
+This is the cost sustained by the RTB for the contextual path traffic. Please refer to the seller side discussion for details.
+
+### F: Extra cost of running the Buyer RTB Server
+
+There will be changes both in the RTB processing of the PAS request and in the size of the data to be set to the RTB server. We assume this difference in cost to be negligible.
+
+### L: Cost of running the BFE
+
+This is the [VM Cost][12] marked C2 and C5 in the B&A explainer. With PAS, the BFE is running an additional flow to PA, thus incurring extra cost. The cost of the PAS flow could be considered equivalent to the cost of processing an extra CustomAudience (or Interest Group in the Chrome parlance).
+
+### N: Cost of running the Bidding Server
+
+This is where we expect noticeable differences, mostly because of the computation cost. Initial estimates suggest the AppInstall bidding server may require ~70% additional computational power for multiple javascript executions. We should account the extra workload required to run more complex inference
+
+The distribution of the cost for running the Bidding Server based on internal estimates is shown below:
+
+The three factors are:
+
+*   Inference. This is expected to constiture the vast majority of the cost. Details on how to compute the inference costs will be added later.
+*   Cost of preparing the data for ad retrieval: essentially the cost for running the. prepareDataForAdRetrievalUDF. This is a minor contributor.
+*   Base cost for running the bidding process, this is the cost for coordinating the flow, processing the data from the ads retrieval server and sending it to generateBid, handling the response and returning the results to the BFE.
+
+There are a lot of changes in bidding server
```

Comment:
Another big difference is the way generateBid runs - on PA there are separate workers per IG, but PAS there is only 1 worker. Wonder if we know the effect of this on cost, and if so, f we should call it out?

Reply:
good point. added later in the text